lls/arp.c

```diff
@@ -166,6 +199,13 @@ process_arp(struct lls_config *lls_conf, struct gatekeeper_if *iface,
 	case ARP_OP_REQUEST: {
 		uint16_t num_tx;
 
+		/*
+		 * We are reusing the frame, but an ARP reply always goes out
+		 * the same interface that received it. Therefore, the L2
+		 * space of the frame is the same. If needed, the correct
+		 * VLAN tag was set above.
```

Comment:
Once we have a function for the `if` statement above, refer to it here instead of "was set above".

Analogous comment is found twice in `process_nd_neigh_solicitation()`.